editors/code/package.json

```diff
@@ -655,24 +655,30 @@
         "problemMatchers": [
             {
                 "name": "rustc",
+                "owner": "rustc",
+                "source": "rustc",
                 "fileLocation": [
-                    "relative",
+                    "autoDetect",
```

Comment:
Annoyingly the VSCode documentation vs schema says conflicting things about whether this prefers relative->absolute or vice versa, as well as how to capitalize the enum.  In the source (and from my testing) the enum is capitalized as `autoDetect`, and the behavior is it tries a relative path first and falls back to absolute.